### PR TITLE
Tensor type indent fix

### PR DESCRIPTION
**Pull request by mepatrick73**
### Checklist

- [X] Confirmed that `run-checks all` script has been executed.
- [X] Made sure the book is up to date with changes in this PR.

### Related Issues/PRs
#2151 
#2115
Dependant on #2195

### Changes
If a TensorType name is a number, adding _ in front to make it a valid Ident for proc_macro2

### Testing

Ran user model mentionned in #2151

### CODE REVIEW

File: crates/burn-import/src/onnx/op_configuration.rs [hunk 1]
```diff
@@ -768,9 +768,20 @@ pub fn tile_config(node: &Node) -> TileConfig {
...
+        // If the input is not provided, return an empty vector
+        if node.inputs.get(index).is_none() {
+            return Vec::new();
+        }
+
+        match &node.inputs[index].value {
+            Some(Data::Int64s(shape)) => shape.clone(),
+            _ => panic!("Tensor data type must be int64"),
+        }
+    }
```
Comment (antimora) [inline crates/burn-import/src/onnx/op_configuration.rs:781]: Did you intend to include pad related changes? I see you have #2195 open. Perhaps, this is based on that branch and that's why these changes appear?

**Review by antimora — APPROVED** [review]
LGTM other than the pad related changes from #2195

File: crates/burn-import/src/onnx/op_configuration.rs (hunk 1 above)
Comment (mepatrick73) [reply crates/burn-import/src/onnx/op_configuration.rs:781]: Yeah, I had to make the pad related changes in order to test the user model.

**Review by laggui — COMMENTED** [review]
Thanks for the fix 🙂

Changes for the type ident look good to me! But we should remove the pad changes from this PR before merging (especially since the pad PR needs some minor adjustments).

**Review by laggui — APPROVED** [review]
